lib/sarif/brakeman_sarif.rb

```diff
@@ -0,0 +1,48 @@
+module Sarif
+  class BrakemanSarif < BaseSarif
+    BRAKEMAN_URI = 'https://github.com/presidentbeef/brakeman'.freeze
+
+    def initialize(scan_report)
+      super(scan_report)
+      @uri = BRAKEMAN_URI
+      begin
+        @logs = JSON.parse(scan_report.log(''))['warnings']
+      rescue JSON::ParserError
+        @logs = []
+      end
+    end
+
+    def parse_issue(issue)
+      {
+        id: format('BRAKE%<number>.4d', number: issue['warning_code'].to_i),
+        name: "#{issue['check_name']}/#{issue['warning_type']}",
+        level: issue['confidence'].upcase,
```

Comment:
Brakeman confidence levels
```
High
Medium
Weak
```
SARIF levels
```
warning
error
note
none
```

**3.27.10 level property**
A result object MAY contain a property named level whose value is one of a fixed set of strings that specify the severity level of the result.

If present, the level property SHALL have one of the following values, with the specified meanings:

·         "warning": The rule specified by ruleId was evaluated and a problem was found.
·         "error": The rule specified by ruleId was evaluated and a serious problem was found.
·         "note": The rule specified by ruleId was evaluated and a minor problem or an opportunity to improve the code was found.
·         "none": The concept of “severity” does not apply to this result because the kind property (§3.27.9) has a value other than "fail".

I think you'll need to map the brakeman confidence levels to their corresponding SARIF level here

Reply:
The confidence levels are currently converted [to sarif levels.](https://github.com/coinbase/salus/blob/master/lib/sarif/base_sarif.rb#L135) 

However, The current mappings may need to be scaled down such that;
```
HIGH ~> error
Medium ~> warning
Low|Weak ~> note

```